keyboards/split75/split75.h

```diff
@@ -0,0 +1,43 @@
+#pragma once
+
+#include "quantum.h"
+
+inline void led_1_on(void) {writePinHigh(D0);}
+inline void led_2_on(void) {writePinHigh(D1);}
+inline void led_3_on(void) {writePinHigh(D6);}
+
+inline void led_1_off(void) {writePinLow(D0);}
+inline void led_2_off(void) {writePinLow(D1);}
+inline void led_3_off(void) {writePinLow(D6);}
+
```

Comment:
```suggestion
```

Reply:
Why remove this? The code in split75.c is proposed to be removed as well, not sure why. Alternative is that I can have the LED boot animation in my personal keymap. But then I'll need to redefine led_X_on/off again, why not keep them here?

Reply:
You don't need to redefine them, just use the `writePinHigh/Low()` macros, that's what they're for.
IMO LED stuff other than num/caps/scroll indicators should not be hardcoded at the keyboard level, and even then `led_update_user()` allows you to override those so you can implement stuff like layer indicators.